web/src/main/java/org/mskcc/cbio/oncokb/controller/EvidenceController.java

```diff
@@ -222,6 +222,27 @@ ResponseEntity updateVUS(@ApiParam(value = "hugoSymbol", required = true) @PathV
         Set<ClinicalTrial> clinicalTrials = queryEvidence.getClinicalTrials();
         String propagation = queryEvidence.getPropagation();
         
+        List<String> cancerTypes = new ArrayList<>();
+        List<String> subTypes = new ArrayList<>();     
+        Boolean cancerEvidence = true;
+        if(cancerType == null) {
+            cancerEvidence = false;
+        } else {
+            cancerTypes = Arrays.asList(cancerType.split(","));
+            if(subType != null) {
+                subTypes = Arrays.asList(subType.split(","));
+                for(int i = 0;i < subTypes.size();i++) {
+                    if(subTypes.get(i).equals("null")) {
```

Comment:
That was I did before. But by default, split() removes trailing empty strings from result array, which will mess up subTypes. There are ways to turn that mechanism off, but I prefer to pass in a string "null", just want to make sure subTypes is in the original passed in order.